src/rgw/rgw_rados.cc

```diff
@@ -828,16 +828,8 @@ struct complete_op_data {
   AioCompletion *rados_completion{nullptr};
   int manager_shard_id{-1};
   RGWIndexCompletionManager *manager{nullptr};
+  std::function<void(librados::ObjectWriteOperation& o)> bi_update;
```

Comment:
If you don't need to copy it, you might prefer `fu2::unique_function`, since then our functions can include unique_ptrs and other uncopyable stuff.

(`unique_function` will not be making it into C++20 because they couldn't agree on whether to rename it to `any_invocable`, apparently. Sheesh.)